src/main/java/gift/service/TokenService.java

```diff
@@ -11,23 +11,23 @@
 @Service
 public class TokenService {
 
-    private final static SecretKey key = Jwts.SIG.HS256.key().build();
-    private final static int jwtExpirationInMs = 7200000; // 2hour
+    private final static SecretKey KEY = Jwts.SIG.HS256.key().build();
+    private final static int JWT_EXPIRATION_IN_MS = 1000 * 60 * 60 * 2;
```

Comment:
static 키워드가 먼저 오는게 관례입니다
```suggestion
    private static final SecretKey KEY = Jwts.SIG.HS256.key().build();
    private static final int JWT_EXPIRATION_IN_MS = 1000 * 60 * 60 * 2;
```

https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

Reply:
수정하였습니다!